rest-openehr/src/main/java/org/ehrbase/rest/openehr/OpenehrDefinitionQueryController.java

```diff
@@ -121,22 +128,51 @@ public ResponseEntity<QueryDefinitionResponseData> putStoreQuery(
         log.debug("putStoreQuery invoked with the following input: " + qualifiedQueryName + ", version:" + version
```

Comment:
As we are on it, we should get rid of the string concatenation here

Reply:
done